types/react-native-share-extension/index.d.ts

```diff
@@ -0,0 +1,16 @@
+// Type definitions for react-native-share-extension 2.0
+// Project: https://github.com/alinz/react-native-share-extension
+// Definitions by: Haseeb Majid <https://github.com/hmajid230>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+interface ShareData {
+    value: string;
+    type: 'text/plain' | 'images/*';
+}
+
+declare class ShareExtension {
```

Comment:
Can we have `close`?
https://github.com/alinz/react-native-share-extension/blob/master/lib/index.js#L8
https://github.com/alinz/react-native-share-extension/blob/master/examples/Sample1/share.ios.js#L38